.github/ISSUE_TEMPLATE/01-use-package.yml

```
@@ -15,6 +15,7 @@ body:
       options:
       - net8.0-android
       - net9.0-android
+      - net10.0-android
```

Just comparing what we do here:
* https://github.com/dotnet/android/blob/1d4c367e67c181260dba64e5df543d734c5e7e1c/.github/ISSUE_TEMPLATE/01-building-an-application.yml?plain=1#L18
```suggestion
      - net10.0-android (Preview)
```